src/pages/Reservation/ReservUserInfo/ReservUserInfo.js

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import PassengerInfo from './PassengerInfo/PassengerInfo';
+
+export default function ReservUserInfo() {
+  const [bookerName, setBookerName] = useState('');
+  const [bookerEmail, setBookerEmail] = useState('');
+  const [bookerNumber, setBookerNumber] = useState(0);
+
+  const validateName =
+    bookerName.length > 1 || bookerName === '' ? true : false;
+
+  const validateEmail =
+    bookerEmail.includes('@' && '.com') || bookerEmail === '' ? true : false;
+
+  const validateNumber =
+    bookerNumber.toString().length > 9 || bookerNumber === 0 ? true : false;
```

Comment:
```suggestion
  const validateName = bookerName.length > 1 || bookerName === '';
  const validateEmail = bookerEmail.includes('@' && '.com') || bookerEmail === '';
  const validateNumber = bookerNumber.toString().length > 9 || bookerNumber === 0;
```
삼항 연산자 생략해도 결과 같을것 같습니다